sickbeard/naming.py

```diff
@@ -65,10 +65,7 @@ def _is_anime(self):
         Find out if show is anime
         :return: True if show is anime, False if not
         """
-        if self.anime > 0:
-            return True
-        else:
-            return False
+        return bool(self.anime)
```

Comment:
what if self.anime was `-1` before?

The old code returns True, now it returns False.

Is it possible? Was it a bug before?